cli/mender-artifact/main.go

```diff
@@ -215,6 +215,20 @@ func run() error {
 		Action:      Cat,
 	}
 
+	install := cli.Command{
+		Name:        "install",
+		Usage:       "install -m<permissions> <src> <dst>",
```

Comment:
Since this can only copy into the artifact/image, maybe a different usage description would be better. Any suggestions?

Reply:
I think the description covers it.